### ISSUE TITLE
1.47.32 Freezes console upon HEN activation

### ISSUE THREAD

**Issue by Ether009** [aldostools/webMAN-MOD#459]
So as of this version, activating HEN freezes the console. Since webman autostarts upon activating HEN, and there's no way to deactivate that autoloading without tools that themselves require HEN to be active. This effectively results in having to do a system restore.

4.87.1 HFW on a SuperSlim 4004C and HEN 3.0.2

**Comment by aldostools** [comment]
Adding an empty (0 bytes) /boot_plugins.txt to the root of a pendrive FAT32 connected to /dev_usb000 prevent the loading of any plugin.

**Comment by Ether009** [comment]
Tried that. Doesn’t help. It still tries to load webman and freezes.

**Comment by aldostools** [comment]
AFAIK PS3HEN uses /dev_usb000/boot_plugins.txt if it exists. if it is empty, any plugin should be loaded.

**Comment by yunussatriandi** [comment]
https://shopee.co.id/yunussatriandi.ys?v=050&smtt=0.0.3

**Comment by yunussatriandi** [comment]
https://shopee.co.id/yunussatriandi.ys?v=050&smtt=0.0.3

**Comment by yunussatriandi** [comment]
Aksks

**Comment by TheUNkilled123** [comment]
Hey, just wanted to say that I'm having the same error. For me it probably resulted from setting the fan speed to 80%(was just playing around and wanted to see how loud it can get) which froze the console. Now every time the plugin gets loaded it sets the fan speed automatically and semi-freezes the console(nothing works except for the xmb animation in the bacground(and the loading cirle)) I'll try disabling with with the boot_plugins.txt and will report back if I find anything else that could be of use.

**Comment by aldostools** [comment]
@TheUNkilled123 Delete /dev_hdd0/tmp/wm_config.bin then restart to reset wMM settings to the default values.

**Comment by TheUNkilled123** [comment]
Will do that after I'm able to use my file managers/ftp.
I also wanted to ask about the hypervisor squashing the dynamic fan controller.
I thought the issue was still here because LV1 isn't patchable by HEN but while browsing previous webMAN releases I've noticed that one said that it "fixed dynamic fan speed hen bug"? Was that refering to something else?

Also, thanks a lot for your effort and hard work!

**Comment by aldostools** [comment]
Some users continued reporting issues after that "fix". Therefore I recommend to continue using manual fan settings on HEN.

**Comment by TheUNkilled123** [comment]
Ok so, I just deleted the config file and restarted the system. Now after loading the plugin the fan goes to like 30% or something and it's unresponsive to input again. Also even the background and animations are running at like 5 fps. There is no disk activity and the issue is persistant even after another reboot.(Sometimes the console just freezes) So what would be the best way to uninstall without the plugin running?

EDIT: Just found your post here: https://www.psx-place.com/threads/how-to-uninstall-webman-mod.28521/
Will try that and then reinstall(maybe even a lower version?)
Also, do you just recomend the latest or the .27 that I've heard is stable?

EDIT2:Ok so even after a full reinstall the plugin still disables all input and is just stuck on rotating icons :/

**Comment by aldostools** [comment]
Try reinstalling the HFW + HEN again... your issue may not be related to webMAN MOD.

EDIT:
There are hundreds of commits/improvements after .27 (I use 1.47.36 on CFW and it should work fine on HEN too).
But it's up to you to use the version that makes you feel more confortable.

**Comment by TheUNkilled123** [comment]
Yeah I'll try updating the HFW to the latest version along with re installing HEN. I think my prior tinkering with the fan settings and then forcefully powering the console off messed something up since it wants to rebuild the file system at every boot but it just gets stuck at 0% with no disk activity.

I'm gonna go get the latest HFW and hope for the best!

**Comment by TheUNkilled123** [comment]
Ok so update. After many attempts I think I'll have to reformat my drive.
I've reinstalled HEN andd HFW and the file system restore stuck at 0% thing was still there even though I could have booted into the system. Now if I wasn't lazy I would have copied over my saves but instead I continued to play around with the plugin(testing to see if apps still crash when using dynamic fan mode) and when it did "freeze" the webserver was still running and I was able to control the fans and even change files. However I couldn't shut the system down without pulling the plug. After that however I got into the restore file system loop where at every boot it told me that it needed to be restored but it got stuck at 0%. Durin the "restore" there was 0 disk activity. No light indicator or audible disk activity. I've waited 30 mins only for it to get automatically incremented by 1. After that I've tried everything from scanning the drive for bad sectors to attempting to repair them to no avail.
I did find a couple of bad blocks/sectors, but the S.M.A.R.T. data from the drive says that there were no bad/relocated sectors. So it makes me think that the issue was in software and that I shut it off or it crashed at a bad time. I've also tried getting the eid keys but since I'm on HEN it was useless. I guess all I can do now is reformat the drive which sucks because I had a ton of saves on it :/

**Comment by TheUNkilled123** [comment]
And for the final update:
After all the reformats and reinstalls I think I'm good now. No more getting stuck on restoring file system.
Also, I've noticed that some apps(at least multiman) black screens even if the fans are set to manual. So I've tries simply turning off the fan control before entering an app and it works flawlessly. I'm even using the dynamic or auto fan speed as it beats the noise of the manual, while also keeping the system around 60 degs max. So if anyone is having a problem with black screens after setting fan control: **Use the L3+R2+START button combo to disable the fan control, enter your game/app, press ps button to pull up XMB and use the combo again to restart it.** 
Thank you for the great work again aldos!
